diff -ur a/src/g_cvars.cpp b/src/g_cvars.cpp
--- a/src/g_cvars.cpp	UZDoom Source trunk da87fb3d
+++ b/src/g_cvars.cpp	2025-10-28 17:33:18.150096509 -0400
@@ -122,10 +122,10 @@
 
 // [BC] Allow the maximum number of particles to be specified by a cvar (so people
 // with lots of nice hardware can have lots of particles!).
-CUSTOM_CVAR(Int, r_maxparticles, 4000, CVAR_ARCHIVE | CVAR_NOINITCALL)
+CUSTOM_CVAR(Int, r_maxparticles, 2500, CVAR_ARCHIVE | CVAR_NOINITCALL)
 {
 	if (self == 0)
-		self = 4000;
+		self = 2500;
 	else if (self > 65535)
 		self = 65535;
 	else if (self < 100)
diff -ur a/src/common/rendering/hwrenderer/data/hw_cvars.cpp b/src/common/rendering/hwrenderer/data/hw_cvars.cpp
--- a/src/common/rendering/hwrenderer/data/hw_cvars.cpp	UZDoom Source trunk da87fb3d
+++ b/src/common/rendering/hwrenderer/data/hw_cvars.cpp	2025-10-29 01:21:35.594795664 -0400
@@ -56,7 +56,7 @@
 CVAR(Bool, gl_portals, true, 0)
 CVAR(Bool,gl_mirrors,true,0)	// This is for debugging only!
 CVAR(Bool,gl_mirror_envmap, true, CVAR_GLOBALCONFIG|CVAR_ARCHIVE)
-CVAR(Bool, gl_seamless, true, CVAR_ARCHIVE|CVAR_GLOBALCONFIG)
+CVAR(Bool, gl_seamless, false, CVAR_ARCHIVE|CVAR_GLOBALCONFIG)
 
 CUSTOM_CVAR(Int, r_mirror_recursions,4,CVAR_GLOBALCONFIG|CVAR_ARCHIVE)
 {
@@ -115,7 +115,7 @@
 // Texture CVARs
 //
 //==========================================================================
-CUSTOM_CVARD(Float, gl_texture_filter_anisotropic, 16.f, CVAR_ARCHIVE | CVAR_GLOBALCONFIG | CVAR_NOINITCALL, "changes the OpenGL texture anisotropy setting")
+CUSTOM_CVARD(Float, gl_texture_filter_anisotropic, 8.f, CVAR_ARCHIVE | CVAR_GLOBALCONFIG | CVAR_NOINITCALL, "changes the OpenGL texture anisotropy setting")
 {
 	screen->SetTextureFilterMode();
 }
