diff -ur a/source/duke3d/src/config.cpp b/source/duke3d/src/config.cpp
--- a/source/duke3d/src/config.cpp	(terminx)
+++ b/source/duke3d/src/config.cpp	(UpdatedControls)
@@ -613,22 +613,22 @@
 
     static GameControllerButtonSetting const buttons[] =
     {
-        { CONTROLLER_BUTTON_A, gamefunc_Open },
-        { CONTROLLER_BUTTON_B, gamefunc_Inventory },
+        { CONTROLLER_BUTTON_A, gamefunc_Jump },
+        { CONTROLLER_BUTTON_B, gamefunc_Open },
         { CONTROLLER_BUTTON_Y, gamefunc_Quick_Kick },
         { CONTROLLER_BUTTON_BACK, gamefunc_Map },
         { CONTROLLER_BUTTON_LEFTSTICK, gamefunc_Run },
-        { CONTROLLER_BUTTON_RIGHTSTICK, gamefunc_Toggle_Crouch },
-        { CONTROLLER_BUTTON_DPAD_UP, gamefunc_Previous_Weapon },
-        { CONTROLLER_BUTTON_DPAD_DOWN, gamefunc_Next_Weapon },
-        { CONTROLLER_BUTTON_LEFTSHOULDER, gamefunc_Crouch },
-        { CONTROLLER_BUTTON_RIGHTSHOULDER, gamefunc_Alt_Fire },
+        { CONTROLLER_BUTTON_RIGHTSTICK, gamefunc_Center_View },
+        { CONTROLLER_BUTTON_DPAD_UP, gamefunc_AutoRun },
+        { CONTROLLER_BUTTON_DPAD_DOWN, gamefunc_Inventory },
+        { CONTROLLER_BUTTON_LEFTSHOULDER, gamefunc_Previous_Weapon },
+        { CONTROLLER_BUTTON_RIGHTSHOULDER, gamefunc_Next_Weapon },
         { CONTROLLER_BUTTON_MISC, gamefunc_Third_Person_View },
     };
 
     static GameControllerButtonSetting const buttonsDuke[] =
     {
-        { CONTROLLER_BUTTON_X, gamefunc_Inventory },
+        { CONTROLLER_BUTTON_X, gamefunc_Toggle_Crouch },
         { CONTROLLER_BUTTON_DPAD_LEFT, gamefunc_Inventory_Left },
         { CONTROLLER_BUTTON_DPAD_RIGHT, gamefunc_Inventory_Right },
     };
@@ -642,7 +642,7 @@
 
     static GameControllerDigitalAxisSetting const digitalAxes[] =
     {
-        { CONTROLLER_AXIS_TRIGGERLEFT, 1, gamefunc_Jump },
+        { CONTROLLER_AXIS_TRIGGERLEFT, 1, gamefunc_Alt_Fire },
         { CONTROLLER_AXIS_TRIGGERRIGHT, 1, gamefunc_Fire },
     };
 
