diff -ur a/source/duke3d/src/config.cpp b/source/duke3d/src/config.cpp
--- a/source/duke3d/src/config.cpp	(emileb)
+++ b/source/duke3d/src/config.cpp	(UpdatedControls)
@@ -604,36 +604,38 @@
 
     static GameControllerButtonSetting const buttons[] =
     {
-        { GAMECONTROLLER_BUTTON_A, gamefunc_Open },
-        { GAMECONTROLLER_BUTTON_B, gamefunc_Toggle_Crouch },
-        { GAMECONTROLLER_BUTTON_Y, gamefunc_Quick_Kick },
+        { GAMECONTROLLER_BUTTON_A, gamefunc_Jump },
+        { GAMECONTROLLER_BUTTON_B, gamefunc_Open },
+        { GAMECONTROLLER_BUTTON_X, gamefunc_Toggle_Crouch },
         { GAMECONTROLLER_BUTTON_BACK, gamefunc_Map },
         { GAMECONTROLLER_BUTTON_LEFTSTICK, gamefunc_Run },
-        { GAMECONTROLLER_BUTTON_RIGHTSTICK, gamefunc_Crouch },
-        { GAMECONTROLLER_BUTTON_DPAD_UP, gamefunc_Previous_Weapon },
-        { GAMECONTROLLER_BUTTON_DPAD_DOWN, gamefunc_Next_Weapon },
-        { GAMECONTROLLER_BUTTON_LEFTSHOULDER, gamefunc_Crouch },
-        { GAMECONTROLLER_BUTTON_RIGHTSHOULDER, gamefunc_Alt_Fire },
+        { GAMECONTROLLER_BUTTON_RIGHTSTICK, gamefunc_Center_View },
+        { GAMECONTROLLER_BUTTON_LEFTSHOULDER, gamefunc_Previous_Weapon },
+        { GAMECONTROLLER_BUTTON_RIGHTSHOULDER, gamefunc_Next_Weapon },
         { GAMECONTROLLER_BUTTON_MISC, gamefunc_Third_Person_View },
     };
 
     static GameControllerButtonSetting const buttonsDuke[] =
     {
-        { GAMECONTROLLER_BUTTON_X, gamefunc_Inventory },
+        { GAMECONTROLLER_BUTTON_Y, gamefunc_Quick_Kick },
+        { GAMECONTROLLER_BUTTON_DPAD_UP, gamefunc_AutoRun },
+        { GAMECONTROLLER_BUTTON_DPAD_DOWN, gamefunc_Inventory },
         { GAMECONTROLLER_BUTTON_DPAD_LEFT, gamefunc_Inventory_Left },
         { GAMECONTROLLER_BUTTON_DPAD_RIGHT, gamefunc_Inventory_Right },
     };
 
     static GameControllerButtonSetting const buttonsFury[] =
     {
-        { GAMECONTROLLER_BUTTON_X, gamefunc_Steroids }, // Reload
-        { GAMECONTROLLER_BUTTON_DPAD_LEFT, gamefunc_MedKit },
-        { GAMECONTROLLER_BUTTON_DPAD_RIGHT, gamefunc_NightVision }, // Radar
+        { GAMECONTROLLER_BUTTON_Y, gamefunc_Steroids }, // Reload
+        { GAMECONTROLLER_BUTTON_DPAD_UP, gamefunc_AutoRun },
+        { GAMECONTROLLER_BUTTON_DPAD_DOWN, gamefunc_MedKit },
+        { GAMECONTROLLER_BUTTON_DPAD_LEFT, gamefunc_NightVision }, // Radar
+        { GAMECONTROLLER_BUTTON_DPAD_RIGHT, gamefunc_Third_Person_View },
     };
 
     static GameControllerDigitalAxisSetting const digitalAxes[] =
     {
-        { GAMECONTROLLER_AXIS_TRIGGERLEFT, 1, gamefunc_Jump },
+        { GAMECONTROLLER_AXIS_TRIGGERLEFT, 1, gamefunc_Alt_Fire },
         { GAMECONTROLLER_AXIS_TRIGGERRIGHT, 1, gamefunc_Fire },
     };
 
