diff -ur a/src/common/rendering/v_video.h b/src/common/rendering/v_video.h
--- a/src/common/rendering/v_video.h	Source 20260728-5.0-7b8bea80
+++ b/src/common/rendering/v_video.h	Modified
@@ -75,6 +75,8 @@
 	BACKEND_DEFAULT = VID_BACKEND,
 #elif defined(HAVE_VULKAN) and not defined(__APPLE__)
 	BACKEND_DEFAULT = BACKEND_VULKAN,
+#elif defined(HAVE_GLES2) and not defined(__APPLE__)
+	BACKEND_DEFAULT = BACKEND_OPENGLES,
 #else
 	BACKEND_DEFAULT = BACKEND_OPENGL,
 #endif
