diff -urN a/core/cart_hw/md_cart.c b/core/cart_hw/md_cart.c
--- a/core/cart_hw/md_cart.c	2026-03-27 23:52:19.391581097 -0400
+++ b/core/cart_hw/md_cart.c	2026-03-27 23:55:54.480508903 -0400
@@ -728,6 +728,33 @@
       zbank_memory_map[i].write   = zbank_unused_w;
     }
   }
+  else if (strstr(rominfo.ROMType, "GM") && strstr(rominfo.product, "00000000-00") &&
+            ((rominfo.checksum == 0x2284) && (rominfo.realchecksum == 0x8dcf))) /* Death & Lead */
+  {
+    /* KAISER WAVE board !TIME handler */
+    cart.hw.time_w = mapper_kaiserwave_w;
+
+    /* enable YX5200 hardware */
+    cart.special |= HW_YX5200;
+
+    /* allocate & clear blip buffer for YX5200 audio stream */
+    snd.blips[3] = blip_new(snd.sample_rate / 10);
+    if (snd.blips[3]) blip_clear(snd.blips[3]);
+
+    /* initialize YX5200 audio */
+    yx5200_init(snd.sample_rate);
+
+    /* 64M extension mapping (upper 4MB of flash memory is apparently unused but cartridge board uses same signals as described in https://gitlab.com/doragasu/flatmap64) */
+    for (i=0x40; i<0x80; i++)
+    {
+      m68k.memory_map[i].read8    = NULL;
+      m68k.memory_map[i].read16   = NULL;
+      m68k.memory_map[i].write8   = NULL;
+      m68k.memory_map[i].write16  = NULL;
+      zbank_memory_map[i].read    = NULL;
+      zbank_memory_map[i].write   = NULL;
+    }
+  }
   else if (strstr(rominfo.international, "COLOCODX")) /* ColocoDX */
   {
     /* CFI flash memory OTP area (credit to Stargazer) */
