diff -ur a/es-app/src/guis/GuiMenu.cpp b/es-app/src/guis/GuiMenu.cpp
--- a/es-app/src/guis/GuiMenu.cpp	Source
+++ b/es-app/src/guis/GuiMenu.cpp	Modified
@@ -40,7 +40,7 @@
 #include <memory>
 
 // Música de fondo ES-X (gestor central)
-#include "audio/BackgroundMusicManager.h"
+//#include "audio/BackgroundMusicManager.h"
 
 // Helper local para traducciones .ini y colores de menú
 namespace
@@ -169,16 +169,16 @@
 
 	if (UIModeController::getInstance()->isUIModeFull())
 	{
-		{
-			auto bgm_switch = std::make_shared<SwitchComponent>(mWindow);
-			bgm_switch->setState(BackgroundMusicManager::getInstance().isEnabled());
-
-			s->addWithLabel(_("BACKGROUND MUSIC").c_str(), bgm_switch);
-			s->addSaveFunc([bgm_switch] {
-				BackgroundMusicManager::getInstance().setEnabled(bgm_switch->getState());
-				Settings::getInstance()->setBool("EnableBGM", bgm_switch->getState());
-			});
-		}
+		//{
+			//auto bgm_switch = std::make_shared<SwitchComponent>(mWindow);
+			//bgm_switch->setState(BackgroundMusicManager::getInstance().isEnabled());
+
+			//s->addWithLabel(_("BACKGROUND MUSIC").c_str(), bgm_switch);
+			//s->addSaveFunc([bgm_switch] {
+				//BackgroundMusicManager::getInstance().setEnabled(bgm_switch->getState());
+				//Settings::getInstance()->setBool("EnableBGM", bgm_switch->getState());
+			//});
+		//}
 
 #if defined(__linux__)
 		auto audio_card = std::make_shared< OptionListComponent<std::string> >(mWindow, _("AUDIO CARD").c_str(), false);
